src/genie/libs/parser/iosxr/show_routing.py

```diff
@@ -586,9 +586,10 @@ def cli(self, vrf=None, route=None, protocol=None, output=None):
 
         # [90/15360] via 10.23.90.3, 1w0d, GigabitEthernet0/0/0/1.90
         # [110/2] via 10.1.2.1, 01:50:49, GigabitEthernet0/0/0/3
-        p3 = re.compile(r'^\[(?P<route_preference>\d+)\/(?P<metric>\d+)\] +via +'
-                        r'(?P<next_hop>\S+),( +(?P<date>[\w:]+))?,? +'
-                        r'(?P<interface>[\w\/\.\-]+)$')
+        # [110/2] via 10.1.3.1, 3w3d
+        p3 = re.compile(r'^\[(?P<route_preference>\d+)\/(?P<metric>\d+)\] +'
```

Comment:
thanks for contribution for this fix! can I ask you to add testing for this change by adding device output with parsed output?

reference:
https://pubhub.devnetcloud.com/media/pyats-development-guide/docs/writeparser/writeparser.html#testing-your-parser

Reply:
I have added a test for this.